cedar-drt/fuzz/src/schemas.rs

```diff
@@ -53,6 +58,18 @@ pub fn equivalence_check(lhs: SchemaFragment, rhs: SchemaFragment) -> Result<(),
     }
 }
 
+// A schema fragment is trivial if it is empty or it's the empty namespace with
+// all declarations being empty
+fn is_trivial(schema: &SchemaFragment) -> bool {
+    if schema.0.len() == 0 {
+        true
+    } else if let Some(def) = schema.0.get(&None) {
+        def.entity_types.is_empty() && def.common_types.is_empty() && def.common_types.is_empty()
```

Comment:
`def.common_types.is_empty()` is repeated twice -- is one of them support to be `action_types`?

Reply:
Good catch. Fixing.

Reply:
Fixed.